.github/workflows/postbench.yml

```diff
@@ -0,0 +1,61 @@
+name: PostBenchmarks
+on:
+  workflow_run:
+    workflows: [RunUploadBenchmarks]
+    types: [completed]
+    branches: [master, benchx]
+jobs:
+  on-success:
+    runs-on: ubuntu-latest
+    # if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - run: echo 'The triggering workflow passed'
+      - name: 'Download artifact'
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: context.payload.workflow_run.id,
+            });
+            let matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
+              return artifact.name == "BenchmarkResults"
+            })[0];
+            let download = await github.rest.actions.downloadArtifact({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               artifact_id: matchArtifact.id,
+               archive_format: 'zip',
+            });
+            let fs = require('fs');
+            fs.writeFileSync(`${process.env.GITHUB_WORKSPACE}/BenchmarkResults.zip`, Buffer.from(download.data));
+
+      - name: 'Unzip artifact'
+        run: unzip BenchmarkResults.zip -d .benchmarkci/
+      - name: 'Post results test 1'
+        run: julia -e 'using BenchmarkCI; BenchmarkCI.postjudge()'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: 'Comment on PR test 2'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            let fs = require('fs');
+            let issue_number = Number(fs.readFileSync('./pr_number'));
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              body: 'Thank you for the PR!'
+            });
+      - name: 'Comment on PR test 3'
```

Comment:
What is the difference between test 2 and test 3?

Reply:
They are different approaches of doing the same thing. Since I don't get to push easily to the master to test, I tried to test different approaches in one go.

Reply:
sounds good